build/debian-iptables/Makefile

```diff
@@ -14,33 +14,28 @@
 
 .PHONY:	build push
 
-REGISTRY?="gcr.io/google_containers"
+REGISTRY?="gcr.io/google-containers"
```

Comment:
there was a reason we used _ - some version of docker didn't handle - properly.  Not sure if that is resolved...